test/utils/test-utils.js

```diff
@@ -256,6 +256,15 @@ const uniqueDirectoryForTest = async (assetsPath) => {
     return result;
 };
 
+const getWebpackCliArguments = () =>
+    Object.entries(cli.getArguments()).map((flag) => {
```

Comment:
not available in v4

Reply:
this is only for tests `core-flags` (only fo v5)

Reply:
can we take a param and send filtered flags? doing in every test seems redundant